### DISCUSSION TITLE
Content Analytics Distinct View

### DISCUSSION THREAD

**Discussion by Amee-r** [exoplatform/exo-community#64] [Q&A]
<img width="1107" alt="Screenshot at Jan 29 17-37-06" src="https://github.com/exoplatform/exo-community/assets/157355419/5901b68f-566c-4eed-8298-79f90c5b166d">
Hi all, I have a question on "Number of distinct views / period" at the Content Analytics section.

For example, in the image attached, i'm selecting Jan 1 - Jan 29.

My "Number of distinct views / period" is 18 with +125% increment from the previous period. What is the range of the previous period mentioned here?

**Comment by ahamdi** [comment]
Hi @Amee-r 
The previous period means the last period equal to the period used to parameterize the display of the table
If you are looking for statistics of the today, then it will be the day before
if you are looking at the statistics of a week, then it will be the stats of the week before
etc ...
